### PR TITLE
bug/22685 => UC-Stable

### PR DESCRIPTION
**Pull request by gwbarrett**
If this should not be merged into `UC-Stable`, then just delete this pull-request.

### CODE REVIEW

**Comment by gwbarrett** [comment]
@johnmcmahon: Is there any reason this has not been merged yet?  If we don't want this in `UC-Stable`, then maybe we should just delete this pull request?

**Comment by johnmcmahon** [comment]
@gwbarrett: Actually I think this should go in UC-Stable. There were a few things that made it into this PR that should be there.  I'll comment in those places separately.

File: webpack.config.js [hunk 1]
```diff
@@ -121,8 +121,9 @@ module.exports = {
       'process.env.CLASSIFICATION_BANNER_FOREGROUND': JSON.stringify(process.env.CLASSIFICATION_BANNER_FOREGROUND || 'white'),
       'process.env.CLASSIFICATION_BANNER_TEXT': JSON.stringify(process.env.CLASSIFICATION_BANNER_TEXT || 'UNCLASSIFIED // TESTING & DEVELOPMENT USE ONLY'),
       'process.env.CONSENT_BANNER_TEXT': JSON.stringify(process.env.CONSENT_BANNER_TEXT || '<p>Users must accept the terms and conditions of the User Agreement before signing in to Beachfront. Contact us for account help.</p>'),
-      'process.env.OSM_BASE_URL': JSON.stringify(process.env.OSM_BASE_URL || 'osm.geointservices.io'),
+      'process.env.OSM_BASE_URL': JSON.stringify(process.env.OSM_BASE_URL || 'tile.openstreetmap.org'),
```
Comment (johnmcmahon) [inline webpack.config.js]: Same deal here.

File: src/components/Application.tsx [hunk 2]
```diff
@@ -157,8 +157,14 @@ export class Application extends React.Component<Props, State> {
...
-      this.refreshRecords()
-          .then(this.importJobsIfNeeded.bind(this))
+      this.refreshRecords().then(() => {
+        // Load selected feature if it isn't already (e.g., page refresh w/ jobId).
+        let [jobId] = this.state.route.jobIds
+
+        if (jobId && !this.state.selectedFeature) {
+          this.state.selectedFeature = this.state.jobs.records.find(job => job.id === jobId)
+        }
+      }).then(this.importJobsIfNeeded.bind(this))
```
Comment (johnmcmahon) [inline src/components/Application.tsx]: This fixes the job refresh bug (a different ticket) so i'm surprised this wasn't merged in through a different PR.  Maybe it was merged in master but forgot to be merged into UC-Stable?

It should be fine to leave it, as long as there weren't any other changes related to that fix

**Comment by johnmcmahon** [comment]
Alright, the OSM base url changes have been taken out of master, so after master is merged back into this branch it _should_ be good to merge.  (assuming there haven't been other changes that pop in)

**Comment by gwbarrett** [comment]
@johnmcmahon, I'll let you take a look at this again.

**Pull request merged**